test/address_test.js

```diff
@@ -17,4 +17,14 @@ describe('Address', function () {
             assert.typeOf(element.cep, 'string')
         });
     });
+
+    it('Deve retornar os dados do CEP com formtato unicode no console informando o CEP', async function () {
+        var cep = '01001-000'
```

Comment:
Sugestao: cololocar a variavel cep pra fora do it, e usar ela em todo o arquivo. Criar: valid_cep, algo assim